public/locales/en/translation.json

```diff
@@ -206,9 +214,7 @@
       "text" : {
         "scenariotype" : "Run type"
       }
-    }
-  },
-  "genericcomponent": {
+    },
```

Comment:
This change seems to break some labels (e.g. the "loading lines" labels). Same thing in the FR dict.